src/bridge/cockpitpcpmetrics.c

```diff
@@ -1096,8 +1096,8 @@ my_pmParseUnitsStr(const char *str, pmUnits *out, double *multiplier)
                                               [PM_TIME_MIN]  = 60,
                                               [PM_TIME_HOUR] = 3600 };
         // guaranteed by __pmParseUnitsStrPart; ensure in-range array access
-        assert (dividend.scaleTime >= PM_TIME_NSEC && dividend.scaleTime <= PM_TIME_HOUR);
-        assert (divisor.scaleTime >= PM_TIME_NSEC && divisor.scaleTime <= PM_TIME_HOUR);
+        assert (dividend.scaleTime <= PM_TIME_HOUR);
+        assert (divisor.scaleTime <= PM_TIME_HOUR);
```

Comment:
Hmm.  This change introduces the silent assumption that PM_TIME_NSEC is zero.  This is probably true now and forever, but I still think it reduces readability.  Also, this isn't really our code.  Can we live with the Coverity warning?